#!/usr/bin/env python
PACKAGE = "me5413_world"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed_target", double_t, 0.3, "Default: 0.3[m/s]", 0.3, 0.2, 1.0)
gen.add("PID_Kp", double_t, 1, "Default: 0.2", 0.2, 0, 5.0)
gen.add("PID_Ki", double_t, 1, "Default: 0.05", 0.05, 0, 1.0)
gen.add("PID_Kd", double_t, 1, "Default: 0.1", 0.1, 0, 1.0)

gen.add("purePursuit_DistanceAhead", double_t, 1, "Default: 1.0", 1.0, 0.5, 5.0)

exit(gen.generate(PACKAGE, "path_tracker_pid_purePursuit_node", "path_tracker_pid_purePursuit"))